jvm/src/main/scala/org/json4s/JValue.scala

```diff
@@ -0,0 +1,48 @@
+package org.json4s
```

Comment:
I will move it to `ast`. I think we will end up having to support both versions anyways.